src/ophyd_async/panda/panda.py

```diff
@@ -72,9 +85,12 @@ class PandA(Device):
     pulse: DeviceVector[PulseBlock]
     seq: DeviceVector[SeqBlock]
     pcap: PcapBlock
+    data: DataBlock
 
     def __init__(self, prefix: str, name: str = "") -> None:
         super().__init__(name)
+        if not prefix.endswith(":"):
+            prefix = prefix + ":"
```

Comment:
Not a fan of adding things to the prefix silently. I'd be happy with:
```suggestion
        assert prefix.endswith(":"), f"PandA prefix '{prefix}' must end in ':'"
```
Or even ditch it altogether, as site naming conventions differ a lot and we might change the PandA IOC to have a configurable PV separator when it goes to FastCS...

Reply:
I will do the assert for now. I seem to remember issues on the IOC side if there's no colon at the end

Reply:
Yeah, at the moment give the IOC the prefix of `MY-PANDA-PREFIX` and it makes `MY-PANDA-PREFIX:SEQ1:ENABLE`, so the assert is helpful, but other sites had a different convention so might like `id3{xfg-sd}/seq1/enable` so we want to support that on the IOC side in future and will have to remove the assert when we do that